easy_profiler_converter/converter.h

```diff
@@ -0,0 +1,34 @@
+#ifndef EASY_PROFILER_CONVERTER_H
+#define EASY_PROFILER_CONVERTER_H
+
+///std
+#include<string>
+
+///this
+#include "reader.h"
+
+///nlohmann json
+#include "include/json.hpp"
+
+class JSONConverter EASY_FINAL
+{
+public:
+    JSONConverter(const ::std::string &file_in,
+                  const ::std::string &file_out):
```

Comment:
Not so clear interface. If `file_out` is empty string output will be dumped in `stdout`. May be we should not use classes. Think about more clear interface

Reply:
accepted.